js-1-4/js1-4.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+const title = 
+`===========================
+  現在持っているのタスク一覧
+===========================`;
+const tasks = [
+  { item: '机を片付ける', genre: '掃除'},
+  { item: '牛乳を買う', genre: '買い物'},
+  { item: '散歩する', genre: '運動'},
+];
+
+function answer () {  
+  console.log(title); 
+  tasks.forEach((value, index) => {
+    console.log(`${index} : [内容]${tasks[index].item}` + `、` + `[ジャンル]${tasks[index].genre}`);
```

Comment:
実はこれで大丈夫だったりします ※22行目も同様です。

```suggestion
    console.log(`${index} : [内容]${value.item}` + `、` + `[ジャンル]${value.genre}`);
```